src/view/components/menu/Menu.scss

```diff
@@ -6,7 +6,7 @@
         flex-direction: column;
         background-color: white;
         box-shadow: 0px 3px 9px #00000026;
-        z-index: 5;
+        z-index: 102;
```

Comment:
I don't like this solution. Can you lower the z-index on the button instead?

Reply:
Yes, I lowered the z-index on the button instead